scc/cert/certificate.go

```diff
@@ -20,11 +20,19 @@ func NewCertificate[S Statement](subject S) Certificate[S] {
 	return Certificate[S]{subject: subject}
 }
 
+func NewCertificateWithSignature[S Statement](subject S, signature AggregatedSignature[S]) Certificate[S] {
+	return Certificate[S]{subject: subject, signature: signature}
```

Comment:
Please document that the provided signature is not deep-copied -- or maybe it should, to avoid accidental modification.

Reply:
The signature is indeed deep copied. I will add documentation and test to reflect this.